### PR TITLE
Fixed issue #2289, if the custom seconds to minutes threshold is set bel...

### PR DESCRIPTION
**Pull request by davidkmenta**
...ow 30 seconds than bug occurs

### CODE REVIEW

File: src/test/moment/relative_time.js [hunk 1]
```diff
@@ -42,10 +42,18 @@ test('default thresholds', function (assert) {
 });
 
 test('custom thresholds', function (assert) {
-    // Seconds to minutes threshold
-    moment.relativeTimeThreshold('s', 55);
```
Comment (ichernev) [inline src/test/moment/relative_time.js]: Can you please add a new test instead of modifying an existing one?

File: src/lib/duration/humanize.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ function substituteTimeAgo(string, number, withoutSuffix, isFuture, locale) {
 function relativeTime (posNegDuration, withoutSuffix, locale) {
     var duration = createDuration(posNegDuration).abs();
     var seconds  = round(duration.as('s'));
-    var minutes  = round(duration.as('m'));
+    var minutes  = round(duration.as('m')) || Math.ceil(duration.as('m'));
```
Comment (ichernev) [inline src/lib/duration/humanize.js]: Can you explain the problem in more detail? What is the problem if `duration.as('m') === 0`, or its not `0`, but falsey?

**Comment by ichernev** [comment]
@davidkmenta ping

**Comment by mj1856** [comment]
@davidkmenta ping

**Comment by ichernev** [comment]
Close in favor of #2635.

**Pull request closed without merging**